package.json

```diff
@@ -28,6 +28,8 @@
     "@angular/platform-browser": "~7.2.0",
     "@angular/platform-browser-dynamic": "~7.2.0",
     "@angular/router": "~7.2.0",
+    "@hmcts/frontend": "0.0.39-alpha",
```

Comment:
Latest version is 41 (from the top of my head) and that contains the fixes for the sass issues, so please use the latest version.

Reply:
thanks for pointing out, i have just updated the latest library for both hmcts/frontend and govuk-frontend